### DISCUSSION TITLE
Actions Secrets/Variables have a suboptimal Web GUI (and Secrets/Variables in General)

### DISCUSSION THREAD

**Discussion by joaduo** [community/community#26637] [Actions]
<p>Hi!</p>
<p>Consider the following 2 example Variables (Secrets)</p>
<pre><code class="lang-auto">AWS_ACCESS_KEY_ID = AKIARAY....213
AWS_SECRET_ACCESS_KEY = a8923898u....as09d0a9s
</code></pre>
<p>I need to be able to see the contents of <code>AWS_ACCESS_KEY_ID</code> but not the contents of <code>AWS_SECRET_ACCESS_KEY</code> because the later is the real secret.</p>
<p>Other platforms allow setting when a variable is a secret (masking/hiding them) and when it can be shown in the Web GUI. (for auditing or debugging purposes)<br>
In the example above of AWS you want to know which KEY ID is being used (so you can know which privileges that repo has)</p>
<p>Where can I do a request for this change on the github platform?</p>

**Comment by airtower-luna** [comment, marked as answer]
<aside class="quote no-group" data-username="joaduo" data-post="1" data-topic="252682">
<div class="title">
<div class="quote-controls"></div>
 joaduo:</div>
<blockquote>
<p>Where can I do a request for this change on the github platform?</p>
</blockquote>
</aside>
<p>I think the “Actions and packages” section of the feedback discussions would be the right place:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github/feedback/discussions/categories/actions-and-packages-feedback">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github/feedback/discussions/categories/actions-and-packages-feedback" target="_blank" rel="noopener">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/5814919/181098993-b7211d36-9786-434b-9916-19eed720b429.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/c/9cd3126cb9889371030dd6fca2e6f536b024d865_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/c/9cd3126cb9889371030dd6fca2e6f536b024d865_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/c/9cd3126cb9889371030dd6fca2e6f536b024d865.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/c/9cd3126cb9889371030dd6fca2e6f536b024d865_2_10x10.png"></div>

<h3><a href="https://github.com/github/feedback/discussions/categories/actions-and-packages-feedback" target="_blank" rel="noopener">Discussions · github/feedback</a></h3>

  <p>Public feedback discussions for: GitHub for Mobile, GitHub Discussions, GitHub Codespaces, GitHub Sponsors, GitHub Issues and more! - Discussions · github/feedback</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by joaduo** [comment]
<p>thanks I posted the request at <a href="https://github.com/github/feedback/discussions/17277" class="inline-onebox" rel="noopener nofollow ugc">Feature Request: Secrets GUI, allow masking real secrets and show normal variables · Discussion #17277 · github/feedback · GitHub</a><br>
(in case someone wants to upvote it)</p>
